src/briefcase/platforms/linux/appimage.py

```diff
@@ -44,14 +44,20 @@ def project_path(self, app):
 
     def binary_name(self, app):
         safe_name = app.formal_name.replace(" ", "_")
-        return f"{safe_name}-{app.version}-{self.tools.host_arch}.AppImage"
+        arch = LinuxDeploy.arch(self.tools.host_os, self.tools.host_arch)
+        return f"{safe_name}-{app.version}-{arch}.AppImage"
 
     def binary_path(self, app):
         return self.bundle_path(app) / self.binary_name(app)
 
     def distribution_path(self, app):
         return self.dist_path / self.binary_name(app)
 
+    def verify_tools(self):
+        """Verify LinuxDeploy is available."""
+        super().verify_tools()
+        LinuxDeploy.verify(tools=self.tools)
```

Comment:
Verify LinuxDeploy before building an entire Docker image and then informing the user that LinuxDeploy can't run on ARM.